charts/spire/templates/k8s-workload-registrar-configmap.tpl

```diff
@@ -10,7 +10,7 @@ data:
     trust_domain = "{{ .Values.trustdomain }}"
     # enable when direct socket access to SPIRE Server available:
     # server_socket_path = "/run/spire/sockets/registration.sock"
-    agent_socket_path = "/run/spire/sockets/agent.sock"
+    agent_socket_path = "{{ .Values.agentSocketDir }}/{{ .Values.agentSocketFile }}"
```

Comment:
nit, but would it be a bit cleaner to do nested values instead, may lead to easier to navigate `values.yaml` file as well

`{{ .Values.agent.socketDir }}`

Reply:
ah you do that for oidc, maybe we can follow that pattern